File: backend/common/pkg/utils/is_alpha.go
package utils

import "unicode"

// IsAlpha checks if a string contains only alphabetic characters
func IsAlpha(s string) bool {
	if len(s) == 0 {
		return false
	}
	for _, r := range s {
		if !unicode.IsLetter(r) {
			return false
		}
	}
	return true
}

File: frontend/src/utils/isAlpha.ts
/**
 * Checks if a string contains only alphabetic characters (A-Z, a-z)
 * @param str - The string to check
 * @returns boolean - True if string contains only alphabets, false otherwise
 */
export const isAlpha = (str: string): boolean => {
  if (!str) return false
  return /^[A-Za-z]+$/.test(str)
}
